recipes/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.conf import settings
+
+class Book(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
+    name = models.CharField(max_length=100)
+
+class Ingredient(models.Model):
+    name = models.CharField(max_length=100)
+    note = models.CharField(max_length=1000)
+            
+class Dish(Ingredient):
+    book = models.ManyToManyField('Book')
+    verbose_name = models.CharField(max_length=100)
+    description = models.CharField(max_length=1000)
+    type = models.ForeignKey('DishType')
```

Comment:
type is reserved as python lang word, I prefer to use here dish_type var name, or something else..